### PR TITLE
Update false evaluations

### PR DESCRIPTION
**Pull request by zeroadam**
update false to is_null where needed.

related to https://github.com/friendica/friendica/issues/4020

### CODE REVIEW

File: fromapp/fromapp.php [hunk 1]
```diff
@@ -51,7 +51,7 @@ function fromapp_settings(&$a,&$s) {
 	/* Get the current state of our config variable */
 
 	$fromapp = PConfig::get(local_user(),'fromapp','app');
-	if($fromapp === false)
+	if(is_null($fromapp))
```
Comment (MrPetovan) [inline fromapp/fromapp.php]: Again, please use default values in `Config::get()` call.
````php
$fromapp = PConfig::get(local_user(),'fromapp','app','');
````

File: fromapp/fromapp.php [hunk 2]
```diff
@@ -84,24 +84,29 @@ function fromapp_settings(&$a,&$s) {
...
-        return;
+function fromapp_post_hook(&$a,&$item)
+{
+	if(! local_user()) {
+		return;
+	}
 
-    if(local_user() != $item['uid'])
-        return;
+    if(local_user() != $item['uid']) {
```
Comment (MrPetovan) [inline fromapp/fromapp.php]: Please fix indentation.

File: fromapp/fromapp.php [hunk 3]
```diff
@@ -84,24 +84,29 @@ function fromapp_settings(&$a,&$s) {
...
+		return;
+	}
 
     $app = PConfig::get(local_user(), 'fromapp', 'app');
 	$force = intval(PConfig::get(local_user(), 'fromapp','force'));
 
-    if(($app === false) || (! strlen($app)))
-        return;
+    if(is_null($app) || (! strlen($app))) {
```
Comment (MrPetovan) [inline fromapp/fromapp.php]: Please fix indentation.

File: notimeline/notimeline.php [hunk 4]
```diff
@@ -45,8 +45,9 @@ function notimeline_settings(&$a,&$s) {
 	/* Get the current state of our config variable */
 
 	$notimeline = PConfig::get(local_user(),'system','no_wall_archive_widget');
-	if($notimeline === false)
+	if(is_null($notimeline)) {
```
Comment (MrPetovan) [inline notimeline/notimeline.php]: Please use the default parameter of `Config::get()`:
```php
$notimeline = PConfig::get(local_user(),'system','no_wall_archive_widget',false);
```

File: numfriends/numfriends.php [hunk 5]
```diff
@@ -69,8 +69,9 @@ function numfriends_settings(&$a,&$s) {
 	/* Get the current state of our config variable */
 
 	$numfriends = PConfig::get(local_user(),'system','display_friend_count');
-	if($numfriends === false)
+	if(is_null($numfriends)) {
```
Comment (MrPetovan) [inline numfriends/numfriends.php]: Please use the default parameter of `Config::get()`:
```php
$numfriends = PConfig::get(local_user(),'system','display_friend_count',24);
```

File: qcomment/qcomment.php [hunk 6]
```diff
@@ -49,8 +49,9 @@ function qcomment_addon_settings(&$a,&$s) {
     $a->page['htmlhead'] .= '<link rel="stylesheet"  type="text/css" href="' . $a->get_baseurl() . '/addon/qcomment/qcomment.css' . '" media="all" />' . "\r\n";
 
 	$words = PConfig::get(local_user(),'qcomment','words');
-	if($words === false)
+	if(is_null($words)) {
```
Comment (MrPetovan) [inline qcomment/qcomment.php]: Please use the default parameter of `Config::get()`:
```php
$words = PConfig::get(local_user(),'qcomment','words',t(':-)') . "\n" . t(':-(') . "\n" .  t('lol'));
```

**Comment by zeroadam** [comment]
I knew there were going to be places to use that default, but I wasn't sure exactly!

**Comment by MrPetovan** [comment]
It's a very good parameter, makes the code look cleaner!

**Comment by zeroadam** [comment]
I'll get the hang of them!

**Pull request merged**